src/Form/MultiLinkField.php

```diff
@@ -13,6 +13,25 @@
  */
 class MultiLinkField extends AbstractLinkField
 {
+    private ?int $maximumLinks = null;
+
+    /**
+     * Set the maximum number of links that can be added to this field.
+     */
+    public function setMaximumLinks(int $maximumLinks): static
+    {
+        $this->maximumLinks = $maximumLinks;
+        return $this;
+    }
+
+    /**
+     * Get the maximum number of links that can be added to this field.
+     */
+    public function getMaximumLinks(): ?int
+    {
+        return $this->maximumLinks ?? null;
```

Comment:
```suggestion
        return $this->maximumLinks;
```
No need for `?? null` because null is already null.